packages/manager/apps/container/package.json

```diff
@@ -44,22 +45,27 @@
     "react-use": "^17.3.1"
   },
   "devDependencies": {
+    "@babel/preset-env": "^7.16.11",
+    "@babel/preset-react": "^7.16.7",
     "@ovh-ux/manager-dev-server-config": "^3.0.0 || ^4.0.0",
     "@testing-library/dom": "^8.5.0",
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^12.1.0",
     "@testing-library/user-event": "^13.2.1",
     "@vitejs/plugin-legacy": "^1.3.2",
     "@vitejs/plugin-react-refresh": "^1.3.1",
+    "babel-jest": "^27.5.1",
     "eslint-plugin-react": "^7.20.3",
     "express": "^4.17.1",
     "http-proxy-middleware": "^1.0.4",
+    "jest": "^27.5.1",
     "msw": "^0.35.0",
+    "react-test-renderer": "^17.0.2",
```

Comment:
We use `testing-library`, so adding react-test-render adds a dependency that does the same thing